src/common/timing/nav-timing.js

```diff
@@ -34,30 +34,30 @@ const PNT_TYPES = {
   prerender: 3
 }
 
-export function addPT (offset, pt, v = {}, isLegacy) {
+export function addPT (offset, pt, v = {}, isL1Api = false) {
   if (!pt) return
   v.of = offset
   handleValue(0, v, 'n')
```

Comment:
This `0` needs to be replaced with `v.of` when `value > 0` is enforced to keep 'n' in object.

Reply:
good catch